src/app/shared/runtime-console/fabric8-ui-onlogin.service.ts

```diff
@@ -0,0 +1,22 @@
+import { LoginService } from './../login.service';
+import { OnLogin, OAuthConfigStore } from 'fabric8-runtime-console';
+import { Injectable } from '@angular/core';
+
+@Injectable()
+export class Fabric8UIOnLogin implements OnLogin {
```

Comment:
This class is not correctly implementing OnLogin, it needs a "whenLoggedIn" property.